src/main/java/io/openliberty/tools/intellij/runConfiguration/LibertyRunManagerListener.java

```diff
@@ -40,8 +41,11 @@ public void runConfigurationRemoved(@NotNull RunnerAndConfigurationSettings sett
             try {
                 VirtualFile vBuildFile = VfsUtil.findFile(Paths.get(runConfig.getBuildFile()), true);
                 LibertyModule libertyModule = libertyModules.getLibertyModule(vBuildFile);
-                if (libertyModule != null && libertyModule.getCustomStartParams().equals(runConfig.getParams())) {
-                    libertyModule.setCustomStartParams(""); // clear the custom params
+                if (libertyModule != null) {
+                    libertyModule.setRunInContainer(false); // clear the run in container checkbox
```

Comment:
I think the comment "clear the run in container checkbox" could be confusing, since it is not actually doing anything to the run in container checkbox directly. This is setting the libertyModule to use a regular start instead of a container start.